auto_click_auto/__init__.py

```diff
@@ -1,4 +1,2 @@
-from .core import (
-    enable_click_shell_completion,
-    enable_click_shell_completion_option,
-)
+from .core import enable_click_shell_completion as enable_click_shell_completion
+from .core import enable_click_shell_completion_option as enable_click_shell_completion_option
```

Comment:
The `isort` hook is failing on [CI](https://github.com/KAUTH/auto-click-auto/actions/runs/9669738028/job/26704739804?pr=14#step:5:72)
```suggestion
from .core import (
    enable_click_shell_completion as enable_click_shell_completion,
)
from .core import (
    enable_click_shell_completion_option as enable_click_shell_completion_option,
)
```